pkg/recovery/power_linux.go

```diff
@@ -0,0 +1,32 @@
+package recovery
+
+import (
+	"os"
+	"syscall"
+)
+
+// PowerAction gives poweoff properties
+type PowerAction struct {
+	reboot bool
```

Comment:
I would make `reboot` and `sync` public so they can be set by external packages

Reply:
Makes sense